packages/features/ee/workflows/api/scheduleEmailReminders.ts

```diff
@@ -154,6 +154,7 @@ async function handler(req: NextApiRequest, res: NextApiResponse) {
             html: emailContent.emailBody.html,
             batchId: batchId,
             sendAt: dayjs(reminder.scheduledDate).unix(),
+            replyTo: reminder.booking?.user?.email || senderEmail,
```

Comment:
I think if we reminder.booking?.user?.email isn't there, that is actually an error because we couldn't add the correct  `replyTo`.

So, along with falling back to senderEmail, we should also log this as an error for observation